.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*.{cpp,h}]
+indent_style = space
+indent_size = 4
+max_line_length = 100
+
+[*.dart]
+indent_style = space
+indent_size = 2
+
+[*.{yaml,yml}]
+indent_style = space
+indent_size = 2
```

Comment:
A couple other relevant file types are `json` and `xml`

Reply:
You should also be able to other rules that align with our style as well, i.e., `trim_trailing_whitespace` and `insert_final_newline`